hyperjumptech/hansip-web
# add configuration to determine hansip admin status and display tenant management based on status

## Description

There are features that should only available for hansip admins and not for any other users. One of them is tenant management. The hansip admin status is determined by the role and domain of the user which is configurable in .env variables, like so:
```
NEXT_PUBLIC_HANSIP_DOMAIN=hansip
NEXT_PUBLIC_HANSIP_ADMIN=admin
```
**This configuration should be inline with the configuration in the backend!!!**

...


## Test plan

1. Log in as hansip admin (user with **admin** role in **hansip** domain if you use default configuration), you should see the tenant management menu available in the sidebar menu
2. Log in as non hansip admin, tenant management menu won't be available in the sidebar menu

...
